expression/builtin_miscellaneous_vec.go

```diff
@@ -17,6 +17,7 @@ import (
 	"bytes"
 	"encoding/binary"
 	"fmt"
+	"github.com/pingcap/tidb/util/vitess"
```

Comment:
Thank you for the contribution! I'm not a reviewer on expression, but changes are looking good to me. Just wanted to let you know this line is triggering the linter:
```
import check fail: non-stdlib "github.com/pingcap/tidb/util/vitess" need be group together and after stdlib group in ../../expression/builtin_miscellaneous_vec.go
```

Reply:
Thanks, fixed that import!